### ISSUE TITLE
To clarify, the edge update functionality is only in `msgnet`, not in `schnetpack`, correct?

### ISSUE THREAD

**Issue by sgbaird** [peterbjorgensen/msgnet#1]
From README.md:
> Tensorflow implementation of message passing neural networks for molecules and materials.
The framework implements the [SchNet model](https://arxiv.org/abs/1712.06113) and its extension with edge update network [NMP-EDGE](https://arxiv.org/abs/1806.03146) as well as the model used in [Materials property prediction using symmetry-labeled graphs as atomic-position independent descriptors](https://arxiv.org/abs/1905.06048).
> 
> Currently the implementation does not enable training with forces, but this might be implemented in the future.
> For a more full-fledged implementation of the SchNet model, see [schnetpack](https://github.com/atomistic-machine-learning/schnetpack).
> 
> The main difference between `msgnet` and `schnetpack` is that `msgnet` follows a message passing architecture and can therefore be more flexible in some cases, e.g. it can be used to train on graphs rather than on structures with full spatial information.

**Comment by peterbjorgensen** [comment]
Yes, you are correct, schnetpack does not have the edge update function. The msgnet repository is a bit outdated, because it is using tensorflow 1.x for the neural network part.
At our DTU research team we do have an updated repository, we use internally, using pytorch which also includes the edge update network, but not the voronoi graph generation.
We might release that code if it is of interest and we find the time to do it.
